CHANGES.rst

```diff
@@ -1,8 +1,16 @@
 Release notes
 =============
 
-v1.0.6 2022-xx-xx
+v1.0.6 2022-03-17
 ------------------
+* Generalized 3D description of magnetic SANS
+* Adding a Boucher-type SLD profile to the selection in the model spherical_sld.
+* Turning magic constants to named magic constants
+* Allowing composite mixtures models with multiplicity 
+* Improving superball model
+* Bugfix in Magnetic SLD calculation
+* A number of docunentation updates
```

Comment:
I think I would change `Allowing`, `Improving` etc into past tense, since these were things done in this release.
E.g.
`Changed magic constants into named constants`
`Composite mixture models with multiplicity are now allowed`
`Improved superball model`
`Fixed a bug in magnetic SLD calculations`
`Updated documentation`